lib/scancodeMap.js

```diff
@@ -1480,7 +1491,7 @@ module.exports = new Map([
   ['ncgl-uk-2.0', 'NCGL-UK-2.0'],
   ['ncsa-httpd-1995', 'LicenseRef-scancode-ncsa-httpd-1995'],
   ['nero-eula', 'LicenseRef-scancode-nero-eula'],
-  ['net-snmp', 'Net-SNMP'],
+  ['net-snmp', 'LicenseRef-scancode-net-snmp'],
```

Comment:
Because this is going from mapping to an SPDX license to mapping to a LicenseRef, I checked this at [SPDX License List](https://spdx.org/licenses/).  `Net-SNMP` is on the SPDX list of licenses. I wonder why it got mapped to a LicenseRef.

The before mapping to `Net-SNMP` was added in the original PR #322 adding this file.

Reply:
I added an issue to Scancode... https://github.com/aboutcode-org/scancode-licensedb/issues/59

Reply:
@qtomlinson Would it be ok for us to make a manual update to this PR before merging?

```suggestion
  ['net-snmp', 'Net-SNMP'],
```

Reply:
@elrayle, I think it should be ok for us to correct it.